packages/itwinui-react/src/core/Tree/TreeNode.tsx

```diff
@@ -15,23 +15,47 @@ type TreeNodeProps = {
    * It has to be compatible with HTML id attribute.
    */
   nodeId: string;
+  /**
+   * Props for treeNode
+   */
+  nodeProps?: React.ComponentProps<'div'>;
   /**
    * The main text displayed on the node.
    */
   label: React.ReactNode;
+  /**
+   * Props for TreeNode label(effects both the main and sub label).
+   */
+  labelProps?: React.ComponentProps<'div'>;
+  /**
+   * Props for the TreeNode's main label.
+   */
+  titleProps?: React.ComponentProps<'div'>;
   /**
    * Small note displayed below main label.
    */
   sublabel?: React.ReactNode;
+  /**
+   * Props for TreeNode sublabel
+   */
+  sublabelProps?: React.ComponentProps<'div'>;
   /**
    * Icon shown before label and sublabel content.
    */
   icon?: JSX.Element;
+  /**
+   * Props for TreeNode Icon
+   */
+  iconProps?: React.ComponentProps<'span'>;
   /**
    * Flag whether the node has child sub-nodes. It is used to show expander icon.
    * @default false
    */
   hasSubNodes?: boolean;
+  /**
+   * Props for subNode
+   */
+  subNodeProps?: React.ComponentProps<'ul'>;
```

Comment:
More appropriate name: `subNodeProps` → `subTreeProps`

Also update the jsdoc to more clearly describe which element this affects.

Reply:
Changed in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/779fc89a58412cd969726940732db0cb1ead7920).